### PR TITLE
Deployment tests for Jenkins, Java7,8

### PR DESCRIPTION
**Pull request by jerjou**
This creates a jenkins script to be run nightly, which actually deploys the std app and runs the selenium tests on them. Since the app is deployed, we're not able to fully test the auth flow without a Real Account, but we do as much as we can.

This also deploys to runtime:java8, after deploying to normal GAE std.

### CODE REVIEW

**Comment by jerjou** [comment]
(this should also fix the jenkins tests, after we change the jenkins config to run `jenkins.sh` instead of `travis.sh`)

File: bookshelf-standard/3-binary-data/pom.xml [hunk 1]
```diff
@@ -116,8 +116,8 @@ Copyright 2016 Google Inc.
 
     <dependency>                        <!-- Google Cloud Client Library for Java -->
       <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud</artifactId>
-      <version>0.5.1</version>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>0.10.0-beta</version>
```
Comment (lesv) [inline bookshelf-standard/3-binary-data/pom.xml]: 0.11.0 was just released.

File: bookshelf-standard/4-auth/pom.xml [hunk 2]
```diff
@@ -116,8 +116,8 @@ Copyright 2016 Google Inc.
 
     <dependency>                        <!-- Google Cloud Client Library for Java -->
       <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud</artifactId>
-      <version>0.5.1</version>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>0.10.0-beta</version>
```
Comment (lesv) [inline bookshelf-standard/4-auth/pom.xml]: 0.11.0 was just released.

Comment (jerjou) [inline bookshelf-standard/4-auth/pom.xml]: Huh... I don't see it in maven central yet.. propagation delay?

http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.google.cloud%22%20AND%20a%3A%22google-cloud%22

**Pull request merged**